sym_api_client_python/clients/sym_bot_client.py

```diff
@@ -51,6 +54,16 @@ def __init__(self, auth, config):
         self.async_agent_session = None
         self.bot_user_info = None
         self.health_check_client = None
+        #Required for aiohttp to use truststore
+        if self.config.data[_TRUSTSTORE_PATH]:
+            logging.debug("Setting truststorePath for async calls to {}".format(config.data[_TRUSTSTORE_PATH]))
+            self.async_ssl_context = ssl.create_default_context(cafile=self.config.data[_TRUSTSTORE_PATH])
+        else: 
+            logging.debug("Setting truststore for async calls to system truststore")
+            self.async_ssl_context = ssl.create_default_context()
+        #Required to work around a bug in Python 3.8+ on Windows. Can be removed once fixed in aiohttp
+        if sys.version_info[0] == 3 and sys.version_info[1] >= 8 and sys.platform.startswith('win'):
```

Comment:
is there a risk that we are changing the loop policy too late? like if asyncio has already been initialized before?

I have mixed feelings on adding this in the BDK, maybe it would be safer to use it as a workaround (in the bot) when you are facing the issue?
I'm thinking for instance what if in some context setting this loop policy is the wrong thing to do?

Reply:
I am open to taking that out and putting it in the README.md

Reply:
@anthony-symphony Yes please I would be more comfortable with this specific workaround out of the code base
Do our customers use Windows for production or only for development?